README.md

```diff
@@ -35,7 +35,26 @@ of IRATI in future releases.
 
 ## 2. Build instructions
 
-### 2.1. Building on Ubuntu 16.04, Debian 8 and Debian 9
+Build Status of the Master branch on Debian via Gitlab Runner.
+
+[![pipeline status](https://gitlab.com/arcfire/irati/stack/badges/master/pipeline.svg)](https://gitlab.com/arcfire/irati/stack/commits/master)
+
+Build Status of arcfire branch on Debian via Gitlab Runner.
+
+[![pipeline status](https://gitlab.com/arcfire/irati/stack/badges/arcfire/pipeline.svg)](https://gitlab.com/arcfire/irati/stack/commits/arcfire)
+
+Build Status of the Master branch on Ubuntu via Travis CI.
+
+[![Build Status of the Master branch on Ubuntu](https://travis-ci.org/IRATI/stack.svg?branch=master)](https://travis-ci.org/IRATI/stack)
+
+Build Status of arcfire branch on Ubuntu via Travis CI.
+
+[![Build Status of arcfire branch on Ubuntu](https://travis-ci.org/IRATI/stack.svg?branch=arcfire)](https://travis-ci.org/IRATI/stack)
+
+
+
+### Building on Ubuntu 16.04 and Debian 8, Debian 9
+#############################################################################
```

Comment:
Please remove this line, as it does not render in a nice way.

Reply:
Ahh yes too true, fixed that up now.